packages/cli-repl/test/e2e-auth.spec.ts

```diff
@@ -269,7 +269,11 @@ describe('Auth e2e', function() {
             'db.dropAllUsers()'
           );
           await eventually(async() => {
-            shell.assertContainsOutput('{ n: 2, ok: 1 }');
+            try {
+              shell.assertContainsOutput('{ n: 2, ok: 1 }');
```

Comment:
Question: is that caused by a version difference that at some point the server returns `n` as a long now? 🤔

Reply:
Yes, it does (on 5.0+) – good that you point this out, I’ll add a comment :+1: